base/linalg/generic.jl

```diff
@@ -514,11 +539,29 @@ julia> norm(v)
 julia> norm(v, Inf)
 6.0
 ```
+"""
+norm(x::AbstractVector, p::Real=2) = vecnorm(x, p)
+
+"""
+    norm(A::AbstractMatrix, p::Real=2)
 
 For matrices, the matrix norm induced by the vector `p`-norm is used, where valid values of
```

Comment:
You're absolutely correct! I had only induced matrix norms in mind. Either induced norm or operator norm seems more precise than matrix norm for these methods. Thanks for setting me right! :)